src/Integration.UnitTests/Binding/SolutionRuleSetWriterTests.cs

```diff
@@ -1,130 +0,0 @@
-//-----------------------------------------------------------------------
-// <copyright file="SolutionRuleSetWriterTests.cs" company="SonarSource SA and Microsoft Corporation">
-//   Copyright (c) SonarSource SA and Microsoft Corporation.  All rights reserved.
-//   Licensed under the MIT License. See License.txt in the project root for license information.
-// </copyright>
-//-----------------------------------------------------------------------
-
-using Microsoft.VisualStudio.CodeAnalysis.RuleSets;
-using Microsoft.VisualStudio.TestTools.UnitTesting;
-using SonarLint.VisualStudio.Integration.Binding;
-using System;
-using System.Globalization;
-using System.IO;
-
-namespace SonarLint.VisualStudio.Integration.UnitTests
-{
-    [TestClass]
-    public class SolutionRuleSetWriterTests
-    {
```

Comment:
This is the one that git thinks was deleted when in fact was renamed (See comment above).